data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -102,7 +102,7 @@ then
 			--etcd-ca=/assets/tls/etcd-client-ca.crt \
 			--etcd-metric-ca=/assets/tls/etcd-metric-ca-bundle.crt \
 			--root-ca=/assets/tls/root-ca.crt \
-			--kube-ca=/assets/tls/kube-ca.crt \
+      --kube-ca=/assets/tls/kube-apiserver-complete-server-ca-bundle.crt \
```

Comment:
Small whitespace issue with the merge here

Reply:
Thanks, fixed.  This one is always manual since we moved the entire MCO block earlier